bin/slurm-send-mail.py

```diff
@@ -305,13 +359,20 @@ def runCommand(cmd):
 
 						# create HTML e-mail
 						msg = MIMEMultipart('alternative')
-						msg['subject'] = 'Job %s.%d: %s' % (cluster, jobId, state)
+						msg['subject'] = 'Job %s.%s: %s' % (cluster, job_name, state)
 						msg['To'] = user
 						msg['From'] = emailFromUserAddress
 
 						body = MIMEText(body, 'html')
 						msg.attach(body)
-						s = smtplib.SMTP('localhost')
+                                                # if this server is an SMTP server:
+                                                s = smtplib.SMTP('localhost')
+                                                # Otherwise use a secondary server for example gmail:
+                                                # s = smtplib.SMTP('smtp.gmail.com', 587, timeout=60)
+                                                # s.ehlo()
+                                                # s.starttls()
+                                                # s.ehlo()
+                                                # s.login(USERNAME, APP_PASSWORD)
```

Comment:
I think that it might be useful to add additional configuration options in `conf.d/slurm-mail.conf` to allow the user to define which mail server to use.

I'll add this as a separate issue to be addressed.

Reply:
Now implemented in 1.7 branch.